examples/mcu-board-support/pico_st7789.rs

```diff
@@ -211,7 +211,7 @@ impl slint::platform::Platform for PicoBackend {
                     .map(|point| {
                         let size = DISPLAY_SIZE.to_f32();
                         let position = euclid::point2(point.x * size.width, point.y * size.height)
-                            / window.window.scale_factor().get();
+                            / window.window().scale_factor().get();
```

Comment:
Perhaps `MinimalSoftwareWindow` should `Deref` to `slint::Window` ? Then it also becomes true to its name (instead of `MinimalSoftwareWindowAdapter`)

Reply:
Good idea